shared/chat/conversation/list-hoc.js

```diff
@@ -7,10 +7,10 @@ import {withProps} from 'recompose'
 import type {Props} from './list'
 import type {Options} from './messages'
 
-type OptionsFn = (message: ChatConstants.Message, prevMessage: ChatConstants.Message, isFirstMessage: boolean, isSelected: boolean, isScrolling: boolean, key: any, style: Object, onAction: () => void, isEditing: boolean) => Options
+type OptionsFn = (message: ChatConstants.Message, prevMessage: ChatConstants.Message, isFirstMessage: boolean, isSelected: boolean, isScrolling: boolean, key: any, style: Object, onAction: () => void, onShowEditor: (event: any) => void, isEditing: boolean) => Options
 
 function propsToMessageOptionsFn (props: Props): OptionsFn {
-  return function (message, prevMessage, isFirstMessage, isSelected, isScrolling, key, style, onAction, isEditing = false): Options {
+  return function (message, prevMessage, isFirstMessage, isSelected, isScrolling, key, style, onAction, onShowEditor, isEditing = false): Options {
```

Comment:
You may need to update the call site in `chat/conversation/list.native.js` to reflect this signature change. Bummer flow didn't catch this!